orphedomos-test/orphedomos-test-baseline/pom.xml

```diff
@@ -18,7 +18,7 @@
                 <groupId>org.technologybrewery.orphedomos</groupId>
                 <artifactId>orphedomos-maven-plugin</artifactId>
                 <configuration>
```

Comment:
I (Informational): updating to use the ghcr.io pattern of `<github organization name>/<image name>`